ios/sound.mm

```diff
@@ -24,209 +24,323 @@
 #include "sound.h"
 #include <AVFoundation/AVFoundation.h>
 
+#define kOutputBus 0
```

Comment:
1. Should be in `ios/sound.h`
2. Style should be block caps
3. Naming should be something like `IOS_DEVICE_OUTPUT_BUS` and `IOS_DEVICE_INPUT_BUS`, (if "DEVICE" is appropriate).

Reply:
Hm.  You're following Apple SDK guidelines, not Jamulus ones.  For something like there, where the Apple SDK shouldn't care, the Jamulus style guide should take precedence, I think.

Reply:
OK, we're leaving this as is - we'll work out what to do when clang-format gets involved...